index.html

```diff
@@ -965,7 +965,7 @@
 file '<i>./second_file.ext</i>'
 . . .
 file '<i>./last_file.ext</i>'</pre>
-In the above, <strong>file</strong> is simply the word "file".<br/>
+In the above, <strong>file</strong> is simply the word "file". Straight apostrophes ('like this') rather than curved quotation marks (‘like this’) should be used to enclose the file paths.<br/>
```

Comment:
I would read them much the same - but better to be 100% clear. Thank you @retokromer :)